# Ibai Irastorza Azcarate - Data Scientist

[Email](mailto:batxes@gmail.com) / [Email2](mailto:ibai.irastorza@gmail.com) / [LinkedIn](https://www.linkedin.com/in/ibai-irastorza/) / [GitHub](https://github.com/batxes/) / [Google Scholar](https://scholar.google.es/citations?user=j9TCubcAAAAJ&hl=en)

Ph.D. in Bioinformatics with Computer Engineering background. 10 years of experience in research and data science, and more than 4 in software development. With a diverse professional journey, I've served as a High-Performance Computer Administrator and contributed to the development of various applications in the private sector and academia. Proficient in Python, Pandas and Pyplot, I am confident in handling large genomic data. I am also committed to continuous learning, evident in my recent participation in Deep Learning workshops or courses in Machine learning in weather and climate. 
I excel in synthesizing information into compelling theses and research articles, showcasing strong written communication skills. My international academic experience has cultivated cultural awareness, adaptability, and resilience, enabling effective collaboration in diverse teams. Proficient in English, I have attended numerous international conferences to showcase my work, fostering confidence in my English communication and presentation abilities. Additionally, I have provided supervision to colleagues and successfully managed substantial projects, affirming my proficiency in leadership and systematic skills.

## üéì Education
- **Ph.D., Bioinformatics** | Complutense University of Madrid (Oct 2013 - May 2018)		  		
- **M.Sc., Bioinformatics**	| National Health Institute Carlos III, Madrid (Oct 2012 - Oct 2013)	 			        		
- **B.Sc., Computer Engineering** | Complutense University of Madrid (Sep 2009 - Sep 2011)
- **B.Sc., Technical Computer Engineering in Computer Systems** | University of Mondragon, Spain (Jul 2003 - Jul 2009)

## üíº Work experience 
**Post-Doctoral Researcher @ <a href="https://www.mdc-berlin.de/pombo#t-team" target="_blank">MDC-BIMSB (Berlin)</a>** (Jun 2018 - Present)
- Characterization of genomic structural differences between the homologous chromosomes of mouse embryonic stem cells
- Analysis, interpretation and integration of spatial genomics data with epigenetic data 

**Ph.D. Candidate @ <a href="https://www.cabd.es/en/research_groups/microbiology-s-platypus/lab-members---collaborators-254.html" target="_blank">CABD (Seville) / Complutense University of Madrid </a>** (Oct 2013 - May 2018)
- Integrative modeling of the 3D structure of macromolecules
- 3D modeling of proteins, proteins complexes and chromatin
- Analysis, interpretation and integration of spatial genomic data, proteomic data and epigenetic data 

**M.Sc. Internship @ <a href="https://www.cos.uni-heidelberg.de/en" target="_blank">University of Heidelberg and the Center for Organismal studies, Heidelberg, Germany</a>** (May 2013 - Sep 2013)
- Integrative 3D modeling of protein complexes
- Analysis, interpretation and integration of spatial proteomic data

**Android Applications Developer** (Oct 2011 - May 2013)
- Game and application development as a freelancer

**Software Developer @ <a href="https://www.accenture.com/es-es" target="_blank">Accenture / BBVA (Madrid)</a>** (Oct 2011 - Aug 2012)
- Development of different applications. Maintenance and support of bank applications

**Software Developer @ <a href="https://www.dnb.com/business-directory/company-profiles.mondigital_sl.10ff1802ff897194db0d783759ce1911.html" target="_blank">Mon Digital (IT company, Madrid)</a>** (May 2011 - Oct 2011)
- Design and development of web sites using Python, Javascript and css.

**Software Developer @ <a href="https://ses.jrc.ec.europa.eu/eirie/en/etic-embedded-technologies-innovation-center-s-coop-1" target="_blank"> Embedded Technologies Innovation Centre / Microsoft (Arrasate, Spain)</a>** (Sep 2008 - Jul 2009)
- Software development in .Net Micro Framework

## üñã Courses/Activities

**Massive Open Online Course from ECMWF in Machine learning in Weather and Climate @ <a href="https://learning.ecmwf.int/" target="_blank"> ECMWF </a>** (Dec 2023 - Present) <a href=" https://github.com/batxes/Machine_Learning_in_Weather_and_Climate" target="_blank"> My Code </a>

**Course in Machine Learning @ <a href="https://github.com/DataTalksClub/machine-learning-zoomcamp/tree/master/" target="_blank"> DataTalksClub </a>** (Sep 2023 - Dec 2023) <a href="https://github.com/batxes/MLzoomcamp" target="_blank"> My Code </a>

**Summer School on Applied Deep Learning in Bioinformatics @ <a href="https://rth.dk/media/SummerSchool2023/Poster_SummerSchool2023_v04.pdf" target="_blank"> Copenhagen University </a>** (Aug 2023)

**Review Editor in <a href="https://www.frontiersin.org/journals/cell-and-developmental-biology/editors" target="_blank">‚ÄúFrontiers in Cell and Developmental Biology‚Äù</a>** (Aug 2023 - Present)

**Master Thesis reviewer of Altair Hern√°ndez Chichilla, Pompeu Fabra University, Barcelona** (Jul 2020)

**Master degree Teacher in Bioinformatic Analysis. <a href="https://www.upo.es/formacionpermanente/master-formacion-permanente/analisis-bioinformatico-avanzado/" target="_blank">Universidad Pablo de Olavide (UPO), Sevilla.</a>** (Jan 2016 - Mar 2018) R and Perl programming languages  

**Integration of MultiOmic Data in Genetic Association Studies. Universidad de Sevilla.** (Sep 2017)

**High Performance Computer Administrator @ CABD (Seville)** (Dec 2015 - Dec 2017)

**Secondment @ <a href="https://salilab.org/" target="_blank">University California San Francisco</a>** (Sep 2014 - Oct 2014)

**3D genome modeling workshop, Strasbourg** (Sep 2014)

## üíª Technical Skills:

- Python (Pandas, Numpy, Pyplot, Seaborn ‚Ä¶), Bash, Git, Inkscape, Docker 
- Experimenting with Tensorflow, Keras, Flask, AWS lambda and Kubernetes.  
- Also worked with Java, C, C#, Javascript, CSS, Html, PHP, Corba, Jinja, Django, Werkzeug, MySQL, SQLite, PostgreSQL.
- Software engineering, UML and different development states of informatic projects.
- Experience in development environments like NetBeans, Eclipse, Visual Studio and Visual Code.
- Functionality and structure of Operatives systems based on Linux.
- Creation and management of Network protocols.
- Web positioning (SEO) knowledge.
- .Net Framework, MS Office Suite, Libre Office. 

## üé§ Public Speaking in International meetings

1.  CABD 20th scientific anniversary, Sevilla (Oral Communication) (06/06/2023) 
2.  SPP2202 monthly meeting (Oral Communication, Online) (10/05/2023) 
3.  Joint Meeting in Neuroepigenetics, Alicante (Organization and Oral Communication) (31/03/2023) 
4.  4DN monthly meeting (Oral Communication, Online) (06/03/2023) 
5.  Annual lab retreat, Berlin (Oral Communication) (21/11/2022) 
6.  Till Bartke Lab retreat, Berlin (Oral Communication, invited speaker) (21/09/2022) 
7.  Bioinformatics Social Meetings, Berlin (Oral Communication, Online) (27/01/2021) 
8.  4D Epigenome conference, Venice (02/10/2019) 
9.  4DN-ASCB Satellite Meeting, San Diego (Oral Communication) (06/12/2018) 
10.  4DN Annual Meeting, San Diego (04/12/2018) 
11.  Annual lab retreat, Berlin (Oral Communication) (21/11/2018) 
12.  3D/4D organization of chromatin, BioMed Conference, Barcelona. (13/11/2017) 
13.  ZENCODE/DevCom Conference, Carmona (27/03/2017) 
14.  4D Genome ERC Workshop. Dynamics of Genome Structure, Barcelona. (22/09/2016) 
15.  XIII Symposium on Bioinformatics. Valencia, Spain. (Oral Communication, Highlight Talk) (10/05/2016) 
16.  HPC admin tech 2016 conference, Madrid. (02/03/2016) 
17.  IX meeting of the Protein Structure and Function Network, Sevilla (Oral Communication) (11/11/2015) 
18.  ISMB/ECCB 2015, Dublin (10/07/2015) 
19.  13th European Conference on Computational Biology, Strasburg (07/09/2014) 
20.  V IMMPC Annual Conference and 4DCellFate Workshop, Barcelona (Oral Communication) (27/03/2014) 

## üèÜ Accomplishments

**FEBS long term fellowship** [https://www.febs.org/funding/fellowships/](https://www.febs.org/funding/fellowships/) _(Jun 2018 - Dec 2020)_ <br>


## üí¨ Languages

üá™üá∫ **Euskara/Basque**: Native <br>
üá∏üáµ **Spanish**: Native <br>
üá∫üá∏ **English**: C1 Proficient User <br>
üá©üá™ **German**: A2 Basic User

## üî¨ Research Publications

0. Ibai Irastorza-Azcarate,#, Alexander Kukalev,#, Rieke Kempfer,#, Christoph Thieme, Guido Mastrobuoni, Julia Markowski, Gesa Loof, Thomas M. Sparks, Emily Brookes, Kedar Natarajan, Mario Nicodemi, Bing Ren, Roland Schwarz, Stefan Kempa, and Ana Pombo. "Extensive folding variability between homologous chromosomes in mammalian cells" In process 

1. RD Acemel, JJ Tena, I Irastorza-Azcarate, F Marl√©taz, C G√≥mez-Mar√≠n, ...  
   "A single three-dimensional chromatin compartment in amphioxus indicates a stepwise evolution of vertebrate Hox bimodal regulation"  
   Nature genetics, 48 (3), 336-341, 2016  
   DOI: <a href="https://doi.org/10.1038/ng.3516">https://doi.org/10.1038/ng.3516</a>

2. W Winick-Ng, A Kukalev, I Harabula, L Zea-Redondo, D Szab√≥, M Meijer, ...  
   "Cell-type specialization is encoded by specific chromatin topologies"  
   Nature, 599 (7886), 684-691, 2021  
   DOI: <a href="https://doi.org/10.1038/s41586-021-03928-5">https://doi.org/10.1038/s41586-021-03928-5</a>

3. A Picco, I Irastorza-Azcarate, T Specht, D B√∂ke, I Pazos, AS Rivier-Cordey, ...  
   "The in vivo architecture of the exocyst provides structural basis for exocytosis"  
   Cell, 168 (3), 400-412. e18, 2017  
   DOI: <a href="https://doi.org/10.1016/j.cell.2016.11.033">https://doi.org/10.1016/j.cell.2016.11.033</a>

4. L Fiorillo, F Musella, M Conte, R Kempfer, AM Chiariello, S Bianco, ...  
   "Comparison of the Hi-C, GAM and SPRITE methods using polymer models of chromatin"  
   Nature methods, 18 (5), 482-490, 2021  
   DOI: <a href="https://doi.org/10.1038/s41592-021-01084-4">https://doi.org/10.1038/s41592-021-01084-4</a>

5. M Franke, E De la Calle-Mustienes, A Neto, M Almuedo-Castillo, ...  
   "CTCF knockout in zebrafish induces alterations in regulatory landscapes and developmental gene expression"  
   Nature communications, 12 (1), 5415, 2021  
   DOI: <a href="https://doi.org/10.1038/s41467-021-25460-z">https://doi.org/10.1038/s41467-021-25460-z</a>

6. C Vicente-Garcia, B Villarejo-Balcells, I Irastorza-Azcarate, S Naranjo, ...  
   "Regulatory landscape fusion in rhabdomyosarcoma through interactions between the PAX3 promoter and FOXO1 regulatory elements"  
   Genome Biology, 18, 1-18, 2017  
   DOI: <a href="https://doi.org/10.1186/s13059-017-1253-4">https://doi.org/10.1186/s13059-017-1253-4</a>

7. R Alcal√°-Vida, J Seguin, C Lotz, AM Molitor, I Irastorza-Azcarate, A Awada, ...  
   "Age-related and disease locus-specific mechanisms contribute to early remodelling of chromatin structure in Huntington‚Äôs disease mice"  
   Nature communications, 12 (1), 364, 2021  
   DOI: <a href="https://doi.org/10.1038/s41467-020-20842-7">https://doi.org/10.1038/s41467-020-20842-7</a>

8. I Irastorza-Azcarate, RD Acemel, JJ Tena, I Maeso, JL G√≥mez-Skarmeta, ...  
   "4Cin: A computational pipeline for 3D genome modeling and virtual Hi-C analyses from 4C data"  
   PLoS Computational Biology, 14 (3), e1006030, 2018  
   DOI: <a href="https://doi.org/10.1371/journal.pcbi.1006030">https://doi.org/10.1371/journal.pcbi.1006030</a>

9. J Markowski, R Kempfer, A Kukalev, I Irastorza-Azcarate, G Loof, B Kehr, ...  
    "GAMIBHEAR: whole-genome haplotype reconstruction from Genome Architecture Mapping data"  
    Bioinformatics, 37 (19), 3128-3135, 2021  
    DOI: <a href="https://doi.org/10.1093/bioinformatics/btab354">https://doi.org/10.1093/bioinformatics/btab354</a>

10. I Irastorza-Azcarate, D Casta√±o-D√≠ez, DP Devos, O Gallego  
    "Live-cell structural biology to solve biological mechanisms: The case of the exocyst"  
    Structure, 27 (6), 886-892, 2019  
    DOI: <a href="https://doi.org/10.1016/j.str.2019.03.009">https://doi.org/10.1016/j.str.2019.03.009</a>
